src/function.js

```diff
@@ -0,0 +1,70 @@
+export default function showAllTodo(tasks) {
+  const listElement = document.querySelector('.list');
+  listElement.innerHTML = '';
+
+  tasks.list.forEach((listItem) => {
+    listElement.innerHTML += `<div class="row" id="${listItem.index}" >
+            <div class="input">
+            <input type="checkbox" class="check" ${
+  listItem.completed ? 'checked' : ''
+}>
+            <input class="list-text" value="${listItem.description}" readonly>
+            </div>
+            <div class="icon">
+            <i class="fas fa-ellipsis-v"></i>
+            <i class="far fa-trash-alt hidden"></i>
+            </div>
+        </div>
+      `;
+  });
+
+  document.querySelectorAll('.list-text').forEach((inp) => {
```

Comment:
use a full name for variables, I cannot figure out what `inp` stands for.

Reply:
Changed to "listText"